optax/_src/alias.py

```diff
@@ -530,15 +530,49 @@ def adamw(
     *,
     nesterov: bool = False,
 ) -> base.GradientTransformation:
-  """Adam with weight decay regularization.
+  r"""Adam with weight decay regularization.
 
   AdamW uses weight decay to regularize learning towards small weights, as
   this leads to better generalization. In SGD you can also use L2 regularization
   to implement this as an additive loss term, however L2 regularization
   does not behave as intended for adaptive gradient algorithms such as Adam.
```

Comment:
Maybe add "see [Loshchilov et al, 2019]." at the end of this sentence (since now the description is longer and the reader may want to quickly understand these statements).